src/components/Todo/Todo.js

```diff
@@ -1,70 +1,170 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
 import InputItem from '../InputItem/InputItem';
 import ItemList from '../ItemList/ItemList';
 import Footer from '../Footer/Footer';
 import styles from './Todo.module.css';
+import { ItemsContext } from '../../itemsContext';
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
+import update from 'immutability-helper';
+
+const initialItems = [];
+
+const initialCount = initialItems.length;
+
+const saveToLocalStorage = items => {
+  let serialItems = JSON.stringify(items);
+  localStorage.setItem('items', serialItems);
+};
 
 function Todo() {
-  useEffect(() => {console.log('update')});
+  let savedItems, savedCount;
+  if (localStorage.length) {
+    savedItems = JSON.parse(localStorage.getItem('items'));
+    savedCount = parseInt(localStorage.getItem('count'), 10);
+  } else {
+    saveToLocalStorage(initialItems);
+    localStorage.setItem('count', initialCount);
+    savedItems = initialItems;
+    savedCount = initialCount;
+  }
+  useEffect(() => {});
   useEffect(() => {console.log('mount')},[]);
-  const [count, setCount] = useState(4);
-  const [items, setItems] = useState(
-    [
-      {
-        value: 'Сделать урок по React',
-        isDone: false,
-        id: 1
-      },
-      {
-        value: 'Приготовить обед',
-        isDone: false,
-        id: 2
-      },
-      {
-        value: 'Помыть посуду',
-        isDone: false,
-        id: 3
-      },
-      {
-        value: 'Погулять с собакой',
-        isDone: false,
-        id: 4
-      }
-    ]);
+  const [bottomNavigationValue, setBottomNavigationValue] = useState('all');
+  const [count, setCount] = useState(savedCount);
+  const [items, setItems] = useState(savedItems);
+
+  const moveItems = useCallback(
+    (dragIndex, hoverIndex) => {
+      const dragCard = items[dragIndex];
```

Comment:
Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/gh/WebHeroSchool/agatta17_todo/pullRequest?prid=6082343)